pkg/karmadactl/cmdinit/kubernetes/deploy.go

```diff
@@ -124,6 +124,7 @@ func init() {
 // CommandInitOption holds all flags options for init.
 type CommandInitOption struct {
 	ImageRegistry                      string
+	ImagePullPolicy                    corev1.PullPolicy
```

Comment:
Suggest using `string` and transitioning when validating and applying to deployment yaml.

Reply:
thx, updated.